src/main/java/org/aksw/iguana/cc/worker/AbstractWorker.java

```diff
@@ -184,19 +175,9 @@ protected HttpContext getAuthContext(String endpoint) {
     }
 
     public synchronized void addResults(QueryExecutionStats results) {
+        // TODO: check if statement for bugs, if the if line exists in the UpdateWorker, the UpdateWorker fails its tests
```

Comment:
This one can be ignored for now because the UpdateWorker will be rewritten anyway.